test/C/src/Deadline.lf

```diff
@@ -44,6 +45,16 @@ reactor Destination(timeout:time(1 sec)) {
         }
         (self->count)++;
     =}
+
+    reaction(x) {=
```

Comment:
@edwardalee pointed out that the test does not achieve what it aims to test because the two reactions execute in a predefined order since they are in the same reactor. Splitting them into separate reactors, however, would mean they wouldn't share state, making the test impossible. The only way I see this working is using a global variable and a mutex, which is an anti-pattern but could be OK for a test.

Reply:
OK, then the solution should be to just revert the changes to this test.